src/__mocks__/index.tsx

```diff
@@ -0,0 +1,32 @@
+const NOOP = () => {
+  // noop
+};
+
+export enum VideoQuality {
+  QVGA_169 = 'QVGA169',
+  VGA_169 = 'VGA169',
+  QHD_169 = 'QHD169',
+  HD_169 = 'HD169',
+  FHD_169 = 'FHD169',
+  QVGA_43 = 'QVGA43',
+  VGA_43 = 'VGA43',
+  QHD_43 = 'QHD43',
+  HD_43 = 'HD43',
+  FHD_43 = 'FHD43',
+}
+
+export const useMembraneServer = NOOP;
+export const useRoomParticipants = NOOP;
+export const useCameraState = NOOP;
```

Comment:
It would be better if the hooks returned objects with mocked data / functions. Users don't just call the hooks, they also read something from them

Reply:
Done.